test/Microsoft.Azure.WebJobs.Host.UnitTests/PublicSurfaceTests.cs

```diff
@@ -155,9 +155,9 @@ public void WebJobsHostPublicSurface_LimitedToSpecificTypes()
                 "ITriggerBindingStrategy`2",
                 "ConnectionStringNames",
                 "JobHost",
-                "JobHostConfiguration",
```

Comment:
Need to add/update public surface tests for the new assemblies added (WebJobs.Extensions.Storage, WebJobs.Host.Storage) and review the surface area